experiments/config/experiment/real_world_runtimes.yaml

```
@@ -35,7 +35,7 @@ fit_fedeca:
   data_path: "/home/owkin/real-world/"
   # We start on the second url the first is reserved for the server
   urls: ["https://api.org-2.fedeca-rw.cg.owkin.tech", "https://api.org-3.fedeca-rw.cg.owkin.tech", "https://api.org-4.fedeca-rw.cg.owkin.tech", "https://api.org-5.fedeca-rw.cg.owkin.tech", "https://api.org-6.fedeca-rw.cg.owkin.tech", "https://api.org-7.fedeca-rw.cg.owkin.tech", "https://api.org-8.fedeca-rw.cg.owkin.tech", "https://api.org-9.fedeca-rw.cg.owkin.tech", "https://api.org-10.fedeca-rw.cg.owkin.tech", "https://api.org-11.fedeca-rw.cg.owkin.tech"]
```

Is that accurate still? I'm afraid you modified a configuration which is unrelated to the staging env. Either the `urls` should be updated, either the `server_org_id` should persist as it was? 
```suggestion
  # urls: ["https://api.org-2.fedeca-rw.cg.owkin.tech", "https://api.org-3.fedeca-rw.cg.owkin.tech", "https://api.org-4.fedeca-rw.cg.owkin.tech", "https://api.org-5.fedeca-rw.cg.owkin.tech", "https://api.org-6.fedeca-rw.cg.owkin.tech", "https://api.org-7.fedeca-rw.cg.owkin.tech", "https://api.org-8.fedeca-rw.cg.owkin.tech", "https://api.org-9.fedeca-rw.cg.owkin.tech", "https://api.org-10.fedeca-rw.cg.owkin.tech", "https://api.org-11.fedeca-rw.cg.owkin.tech"]
  urls: ["https://substra-backend.org-1.substra-staging.owkin.tech", "https://substra-backend.org-2.substra-staging.owkin.tech", "https://substra-backend.org-3.substra-staging.owkin.tech"]
```